config/env/internal-external.json

```diff
@@ -1,20 +1,21 @@
 {
   "services": {
     "ceramic": {
+      "launchLocation": "jest",
       "mode": "client",
       "apiURL": "https://ceramic-dev-internal.3boxlabs.com"
     },
-    "ceramic2": {
-      "mode": "client",
-      "apiURL": "https://ceramic-dev.3boxlabs.com",
-      "anchorServiceAPI": "https://cas-dev.3boxlabs.com",
-      "ethereumRpc": null,
-      "network": "dev-unstable"
-    },
     "ipfs": {
+      "launchLocation": "ceramic",
       "mode": "none"
     },
+    "ceramic2": {
+      "launchLocation": "jest",
+      "mode": "client",
+      "apiURL": "https://ceramic-dev.3boxlabs.com"
+    },
     "ipfs2": {
+      "launchLocation": "ceramic2",
```

Comment:
Does this actually do anything?  Or is it more of a form of commenting/documenting the behavior?  If the latter, maybe this could be moved into a README or something rather than having configuration options here that don't have a meaningful effect?

Reply:
Agree it's not great because it should really be a comment. Doesn't need to be in README. 

I guess mode = none serves the purpose but I think it may actually be better like

```
ceramic: {
  ipfs: {
    mode: "inprocess"
  }
}
```

thoughts? @stbrody